arangod/Aql/Variable.cpp

```diff
@@ -95,26 +95,34 @@ void Variable::toVelocyPack(VPackBuilder& builder) const {
   }
 }
 
+void Variable::toVelocyPackCommon(velocypack::Builder& builder) const {
```

Comment:
Btw If you write this function between Variable::toVelocyPack(velocypack::Builder& builder) and Variable::toVelocyPack(velocypack::Builder& builder,
                            Variable::WithConstantValue /*tag*/) diff will be smaller